### PR TITLE
Fix v2 event object name

### PR DESCRIPTION
**Pull request by jar-stripe**
### Why?
The SDK uses object strings in the API response to decide what class to parse a resource returned from the Stripe API into.  Recently, the object string for the V2 event base class changed from "event" to "v2.core.event".  We have to update the SDK to match.

### What?
- Changed "event" to "v2.core.event" in _util._convert_to_stripe_object

Verified using the `stripe_webhook_handler.py` example with events triggered from and forwarded by the Stripe CLI

### See Also
http://go/j/DEVSDK-2202

### CODE REVIEW

File: examples/stripe_webhook_handler.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ def webhook():
             # CHECK: fetch_object is present and callable, returning a strongly-typed object (without casting)
```
Comment (xavdid-stripe) [inline examples/stripe_webhook_handler.py:27]: not part of this PR, but this line probably shouldn't be in examples?

**Review by xavdid-stripe — APPROVED** [review]

**Review by prathmesh-stripe — APPROVED** [review]

**Comment by prathmesh-stripe** [comment]
Needs a test to be fixed too.

File: examples/stripe_webhook_handler.py (hunk 1 above)
Comment (jar-stripe) [reply examples/stripe_webhook_handler.py:27]: correct, will fix.  thanks!
